Schemas_draft-03/Collaboration/Viewpoint/bitmap_GET.json

```diff
@@ -0,0 +1,24 @@
+{
+    "type": "object",
+    "properties": {
+        "guid": {
+            "type": "string"
+        },
+        "bitmap_type": {
+            "type": "string",
+            "enum": ["jpg", "png", "bmp"]
```

Comment:
`bmp` should be removed, since we decided not to support bmp for images. [See the BCF XML Bitmap Type enum](https://github.com/BuildingSMART/BCF-XML/blob/master/Schemas/visinfo.xsd#L172)